### ISSUE TITLE
No way to get list of keys from the store

### ISSUE THREAD

**Issue by dipshit** [edegens/ptype#59]
We can get every value and put and delete, but getting the keys themselves is impossible unless the developer stores/remembers them themselves.

**Comment by JackyChiu** [comment]
Could we not get them by asking for all prefixes of the `/store`?

**Comment by dipshit** [comment]
@JackyChiu no: https://github.com/edegens/ptype/blob/master/cluster/store.go#L50

The easiest solution I see is to return a `map[string]string` of keys & values instead of `[]string`
